docs/customizing-evaluation-outputs.md

```diff
@@ -0,0 +1,54 @@
+# Template BPA Customizing Evaluation Outputs
+<a name="note"></a>***Note**: The ARM Template BPA is currently in development. All features that have yet to be implemented have been flagged with an asterisk [\*].*
+
+## Overview
+Template BPA customization is authored in JSON.  Custom evaluation settings are written in a separate configuration and used on tool executing. More details on how to run Template BPA with a configuration file can be found [here](https://github.com/Azure/template-analyzer#using-the-template-bpa). 
+
+## Template BPA Rule Object
+Here are the fields that make up a custom configuration file.
+```javascript
+{
+    "exclusions": { � }, // Parameters that are excluded from the BPA execution.  More details below.
+    "inclusions": { � }, // Only parameters that will be included in the BPA execution.  More details below.
+    "output": { � }, // How the BPA execution output will be displayed to the user.  More details below.
+    "severityReMapping": { � } // Tuple of ruleId and new severity value.  Can change a rule's severity.
+}
```

Comment:
It would make sense to me to support all the BPA parameters inside this file, maybe we could add the file paths here too? Or remove the options that are supported without this file, like the type of output

Reply:
Removed `"output"`